src/Bridges/Constraint/bridges/SplitHyperRectangleBridge.jl

```diff
@@ -174,7 +174,11 @@ end
 
 function MOI.supports(
     model::MOI.ModelLike,
-    attr::Union{MOI.ConstraintPrimalStart,MOI.ConstraintDualStart},
+    attr::Union{
+        MOI.ConstraintPrimalStart,
+        MOI.ConstraintDualStart,
+        MOI.ConstraintDual,
```

Comment:
This is a big dislike from me. ConstraintDual is a result attribute. We shouldn't make bridges "support" it.